app/components/inputs/PathBrowseInput.js

```diff
@@ -0,0 +1,64 @@
+const electron = require("electron");
+const dialog = electron.remote.dialog;
+const mainWindow = electron.remote.getCurrentWindow();
+const ipc = electron.ipcRenderer;
+
+import PathInput from "./PathInput";
+import { PathButton } from "../buttons";
+
+let pathListener;
+
+@autobind
+class PathBrowseInput extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { path: "" };
+  }
+
+  componentDidMount() {
+
+    let self = this;
```

Comment:
I'm pretty sure `@autobind` already covers this as well, so you don't need to store the reference in `self`. Try it out and see if it works.

Reply:
```
PathBrowseInput.js:22 Uncaught TypeError: this.setState is not a function
    at EventEmitter.pathListener (PathBrowseInput.js:22)
    at emitTwo (events.js:106)
    at EventEmitter.emit (events.js:194)
```
I'll keep it how it is

Reply:
ok